ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.view.js

```diff
@@ -13,210 +13,316 @@
+                        {
+                            data.showRelevanceScore ? 
+                            <div className="detail-custom details-property" data-help={`Relevance: ${data.relevance}`} title={`Relevance: ${data.relevance}`}>
+                                <span>
+                                {data.roundedRelevance}
+                                </span>
+                            </div> : ''
+                        }
+                        {
+                            data.showSource ? 
+                            <div className="detail-source details-property" title={`${HandleBarsHelpers.getAlias('source-id')}: ${data.metacard.properties['source-id']}`} 
+                                data-help={HandleBarsHelpers.getAlias('source-id')}>
+                                {
+                                    data.local ? 
+                                    <React.Fragment>
+                                        <span className="fa source-icon fa-home">
+                                        </span>
+                                        <span>local</span>
+                                    </React.Fragment> : 
+                                    <React.Fragment>
+                                        <span className="fa source-icon fa-cloud">
+                                        </span>
+                                        <span>
+                                            {data.metacard.properties['source-id']}
+                                        </span>
+                                    </React.Fragment>
                                 }
-                                break;
+                            </div> : ''
+                        }
+                    </div>
+                    <div className="content-footer">
+                        <div className="result-validation">
+                            {
+                                data.hasError ? 
+                                <span className="fa fa-exclamation-triangle resultError" title="Has validation errors." data-help="Indicates the given result has a validation error.
+                                See the 'Quality' tab of the result for more details.">
+                                </span> : ''
+                            }
+                            {
+                                !data.hasError && data.hasWarning ?
+                                <span className="fa fa-exclamation-triangle resultWarning" title="Has validation warnings." data-help="Indicates the given result has a validation warning.
+                                See the 'Quality' tab of the result for more details.">
+                                </span> : ''
+                            }
+                        </div>
+                        <button className="result-download fa fa-download is-button is-neutral" title="Downloads the results associated product directly to your machine." data-help="Downloads
```

Comment:
✏️ results s/b **result's**.  Also I thought we were using "resource" instead of "product"